Vagrantfile

```diff
@@ -0,0 +1,78 @@
+# -*- mode: ruby -*-
```

Comment:
If this is BPF testing VM, IMO should be in `bpf` directory. Keeping it in `/` suggests that this is the main Vagrant that should be used to run Teleport.

Reply:
Done.